app/views/sa/expenses.scala.html

```diff
@@ -33,20 +33,21 @@ <h2 class="margin-top-xlarge">@Messages("sa.expenses.what_expenses_can_i_include
   <p>@Messages("sa.expenses.if_you_do_not_use_simplified_expenses_")</p>
   <p>@Messages("sa.expenses.if_your_business_income_is_below_")</p>
 
-  @components.embed_youtube_video(appConfig.getYoutubeVideoId("expensesForSelfEmployed"), Some(messages("sa.expenses.what_expenses_can_i_include_in_my_sa_tax_return")))
-
-  <p class="padding-top-medium"><a href="@controllers.routes.TranscriptController.onPageLoad("expenses-if-you-are-self-employed")" id="what-expenses-can-i-include-in-my-sa-tax-return-transcript" data-journey-click="link - click:Expenses:Expenses if you are self employed - video transcript">@Messages("sa.expenses.what_expenses_can_i_include_in_my_sa_tax_return") - @Messages("sa.expenses.video_transcript")</a></p>
+<div class="panel panel-border-wide">
+ <p><a href ="https://youtu.be/FjXAH3tMg2I" id="what-expenses-can-i-include-in-my-sa-tax-return-video" target="_blank" data-journey-click="link - click:Expenses:Expenses if you are self employed - video link">@messages("sa.expenses.what_expenses_can_i_include")</a></p>
+  <p><a href="@controllers.routes.TranscriptController.onPageLoad("expenses-if-you-are-self-employed")" id="what-expenses-can-i-include-in-my-sa-tax-return-transcript" data-journey-click="link - click:Expenses:Expenses if you are self employed - video transcript">@Messages("sa.expenses.what_expenses_can_i_include_in_my_sa_tax_return") - @Messages("sa.expenses.video_transcript")</a></p>
+</div>
 
   <h2 class="margin-top-xlarge">@Messages("sa.expenses.claiming_motoring_expenses")</h2>
   <p>@Messages("sa.expenses.you_can_claim_business_expenses_for_")</p>
   <p>@Messages("sa.expenses.you_can_claim_either_the_actual_costs_")</p>
   <p>@Messages("sa.expenses.if_you_decide_to_use_the_actual_costs_")</p>
   <p>@Messages("sa.expenses.the_flat_rate_covers_the_whole_cost_")</p>
 
-  @components.embed_youtube_video(appConfig.getYoutubeVideoId("calculatingMotoringExpenses"), Some(messages("sa.expenses.claiming_motoring_expenses")))
-
-  <p class="padding-top-medium"><a href="@controllers.routes.TranscriptController.onPageLoad("calculating-motoring-expenses")" id="calculating-motoring-expenses-transcript" data-journey-click="link - click:Expenses:Calculating motoring expenses - video transcript">@Messages("sa.expenses.claiming_motoring_expenses") - @Messages("sa.expenses.video_transcript")</a></p>
-
+<div class ="panel panel-border-wide">
+    <p><a href ="https://youtu.be/FjXAH3tMg2I" id="calculating-motoring-expenses-transcript-video" target="_blank" data-journey-click="link - click:Expenses:Calculating motoring expenses - video link">@messages("sa.expenses.view.claiming_motoring_expenses")</a></p>
```

Comment:
This link is incorrect, goes to the wrong video, also the id for it shouldn't say transcript